internal/handlers/commonparams/extract_params.go

```diff
@@ -87,12 +84,6 @@ func ExtractParams(doc *types.Document, command string, value any, l *zap.Logger
 
 		lookup := key
 
-		// If the key is the same as the command name, then it is a collection name.
-		// Depending on the driver, the key may be camel case or lower case for a collection name.
```

Comment:
That comment was relevant

Reply:
Thanks for capturing this. Fix made in https://github.com/FerretDB/FerretDB/pull/3494